### PR TITLE
[zh] Translate blog Securing-Admission-Controllers-2022.1.19

### PR DESCRIPTION
**Pull request by twilight0620**
Fixes: https://github.com/kubernetes/website/issues/32196

Do not have chinese blog:  https://kubernetes.io/blog/2022/01/19/secure-your-admission-controllers-and-webhooks/

so need to translate Securing-Admission-Controllers blog into Chinese

### CODE REVIEW

File: content/zh/blog/_posts/2022-01-19-Securing-Admission-Controllers.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+From the threat model, we developed a set of security best practices that should be adopted to ensure that cluster operators can get the security benefits of admission controllers whilst avoiding any risks from using them.
+-->
+[准入控制](/zh/docs/reference/access-authn-authz/admission-controllers/)和认证、授权都是 Kubernetes 安全性的关键部分。
+Webhook 准入控制器被广泛用于以多种方式帮助提高 Kubernetes 集群的安全性，
+包括限制工作负载权限和确保部署到集群的镜像满足组织安全要求。
+
+然而，与添加到集群中的任何其他组件一样，安全风险也会随之出现。
+一个安全风险示例是没有正确处理准入控制器的部署和管理。
+为了帮助准入控制器用户和设计人员适当地管理这些风险，
+SIG 安全小组[安全文档](https://github.com/kubernetes/community/tree/master/sig-security#security-docs)
```
Comment (tengqm) [inline content/zh/blog/_posts/2022-01-19-Securing-Admission-Controllers.md:40]: SIG 后面跟的那个首字母大写的词不要翻译

```suggestion
SIG Security 的[安全文档](https://github.com/kubernetes/community/tree/master/sig-security#security-docs)小组
```

File: content/zh/blog/_posts/2022-01-19-Securing-Admission-Controllers.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+花费了一些时间来开发一个[准入控制器威胁模型](https://github.com/kubernetes/sig-security/tree/main/sig-security-docs/papers/admission-control)。
+这种威胁模型着眼于由于不正确使用准入控制器而产生的可能的风险，可能允许绕过安全策略，甚至允许攻击者未经授权访问集群。
+
+从威胁模型中，我们开发了一套安全最佳实践，应该采用这些实践来确保集群操作员可以获得准入控制器带来的安全优势，同时避免使用它们带来的任何风险。
```
Comment (tengqm) [inline content/zh/blog/_posts/2022-01-19-Securing-Admission-Controllers.md:44]:
```suggestion
基于这个威胁模型，我们开发了一套安全最佳实践。
你应该采用这些实践来确保集群操作员可以获得准入控制器带来的安全优势，同时避免使用它们带来的任何风险。
```

File: content/zh/blog/_posts/2022-01-19-Securing-Admission-Controllers.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+<!--
+## Admission controllers and good practices for security
+
+From the threat model, a couple of themes emerged around how to ensure the security of admission controllers.
+-->
+
+## 准入控制器和安全的良好做法
+
+从威胁模型中，围绕如何确保准入控制器的安全性出现了几个主题。
```
Comment (tengqm) [inline content/zh/blog/_posts/2022-01-19-Securing-Admission-Controllers.md:54]:
```suggestion
基于这个威胁模型，围绕着如何确保准入控制器的安全性出现了几个主题。
```

File: content/zh/blog/_posts/2022-01-19-Securing-Admission-Controllers.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+* **Admission controller fails closed**. This is a security practice that has a tradeoff, so whether a cluster operator wants to configure it will depend on the cluster’s threat model. If an admission controller fails closed, when the API server can’t get a response from it, all deployments will fail. This stops attackers bypassing the admission controller by disabling it, but, can disrupt the cluster’s operation. As clusters can have multiple webhooks, one approach to hit a middle ground might be to have critical controls on a fail closed setups and less critical controls allowed to fail open.
+* **Regular reviews of webhook configuration**. Configuration mistakes can lead to security issues, so it’s important that the admission controller webhook configuration is checked to make sure the settings are correct. This kind of review could be done automatically by an Infrastructure As Code scanner or manually by an administrator.
+-->
+
+### 安全的 webhook 配置
+
+确保集群中的任何安全组件都配置良好并且准入控制器在这里没有什么不同是很重要的。
```
Comment (tengqm) [inline content/zh/blog/_posts/2022-01-19-Securing-Admission-Controllers.md:69]:
```suggestion
确保集群中的任何安全组件都配置良好是很重要的，在这里准入控制器也并不例外。
```

File: content/zh/blog/_posts/2022-01-19-Securing-Admission-Controllers.md [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+使用准入控制器时需要考虑几个安全最佳实践
```
Comment (tengqm) [inline content/zh/blog/_posts/2022-01-19-Securing-Admission-Controllers.md:70]:
```suggestion
使用准入控制器时需要考虑几个安全最佳实践：
```

File: content/zh/blog/_posts/2022-01-19-Securing-Admission-Controllers.md [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+
+* **为所有 webhook 流量正确配置了 TLS**。
+ API 服务器和准入控制器 webhook 之间的通信应该经过身份验证和加密，以确保处于网络中查看或修改此流量的攻击者无法查看或修改。
```
Comment (tengqm) [inline content/zh/blog/_posts/2022-01-19-Securing-Admission-Controllers.md:73]: 这种 list 的内容都缩进两个空格。

File: content/zh/blog/_posts/2022-01-19-Securing-Admission-Controllers.md [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+ 要实现此访问，API 服务器和 webhook 必须使用来自受信任的证书颁发机构的证书，以便它们可以验证相互的身份。
+* **只允许经过身份验证的访问**。
+如果攻击者可以向准入控制器发送大量请求，他们可能会压倒服务导致其失败。
```
Comment (tengqm) [inline content/zh/blog/_posts/2022-01-19-Securing-Admission-Controllers.md:76]:
```suggestion
如果攻击者可以向准入控制器发送大量请求，他们可能会压垮服务导致其失败。
```

File: content/zh/blog/_posts/2022-01-19-Securing-Admission-Controllers.md [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+确保所有访问都需要强身份验证可以降低这种风险。
+* **准入控制器关闭失败**。
+ 这是一种需要权衡的安全实践，集群操作员是否要对其进行配置取决于集群的威胁模型。
+ 如果一个准入控制器关闭失败，当 API 服务器无法从它得到响应时，所有的部署都会失败。
+ 这可以阻止攻击者通过禁用准入控制器绕过准入控制器，但可能会破坏集群的运行。
+ 由于集群可以有多个 webhook，因此达到中间立场的一种方法是对故障关闭设置进行关键控制，并允许不太关键的控制进行故障打开。
```
Comment (tengqm) [inline content/zh/blog/_posts/2022-01-19-Securing-Admission-Controllers.md:82]:
```suggestion
 由于集群可以有多个 webhook，因此一种折中的方法是对关键控制允许故障关闭，
 并允许不太关键的控制进行故障打开。
```

File: content/zh/blog/_posts/2022-01-19-Securing-Admission-Controllers.md [hunk 9]
```diff
@@ -0,0 +1,137 @@
...
+* **Strictly control external system access**. As a security service in a cluster admission controller systems will have access to sensitive information like credentials. To reduce the risk of this information being sent outside the cluster, [network policies](/docs/concepts/services-networking/network-policies/) should be used to restrict the admission controller services access to external networks.
+* **Each cluster has a dedicated webhook**. Whilst it may be possible to have admission controller webhooks that serve multiple clusters, there is a risk when using that model that an attack on the webhook service would have a larger impact where it’s shared. Also where multiple clusters use an admission controller there will be increased complexity and access requirements, making it harder to secure.
+-->
+
+### 准入控制的安全集群配置
+
```
Comment (tengqm) [inline content/zh/blog/_posts/2022-01-19-Securing-Admission-Controllers.md:99]: 不要大段大段的翻译，对这种列表可以逐条翻译，否则无法 review。请修改。

**Comment by twilight0620** [comment]
I have modified all comments. Thanks for your review.

File: content/zh/blog/_posts/2022-01-19-Securing-Admission-Controllers.md [hunk 10]
```diff
@@ -0,0 +1,156 @@
...
+* **Regular reviews of webhook configuration**. Configuration mistakes can lead to security issues, so it’s important that the admission controller webhook configuration is checked to make sure the settings are correct. This kind of review could be done automatically by an Infrastructure As Code scanner or manually by an administrator.
+-->
+* **定期审查 webhook 配置**。
+  配置错误可能导致安全问题，因此检查准入控制器 webhook 配置以确保设置正确非常重要。
+  这种审查可以由基础设施即代码扫描程序自动完成，也可以由管理员手动完成。
+
+<!--
+### Secure cluster configuration for admission control
+-->
+### 准入控制的安全集群配置
```
Comment (tengqm) [inline content/zh/blog/_posts/2022-01-19-Securing-Admission-Controllers.md:101]:
```suggestion
### 为准入控制保护集群配置
```

File: content/zh/blog/_posts/2022-01-19-Securing-Admission-Controllers.md [hunk 11]
```diff
@@ -0,0 +1,156 @@
...
+In most cases, the admission controller webhook used by a cluster will be installed as a workload in the cluster. As a result, it’s important to ensure that Kubernetes' security features that could impact its operation are well configured.
+-->
+在大多数情况下，集群使用的准入控制器 webhook 将作为工作负载安装在集群中。
+因此，确保正确配置了可能影响其操作的 Kubernetes 安全特性非常重要。
+
+<!--
+* **Restrict [RBAC](/docs/reference/access-authn-authz/rbac/) rights**. Any user who has rights which would allow them to modify the configuration of the webhook objects or the workload that the admission controller uses could disrupt its operation. So it’s important to make sure that only cluster administrators have those rights.
+-->
+* **限制 [RBAC](/zh/docs/reference/access-authn-authz/rbac/) 权限**。
+  任何有权修改 webhook 对象的配置或准入控制器使用的工作负载的用户都可能会中断其操作。
```
Comment (tengqm) [inline content/zh/blog/_posts/2022-01-19-Securing-Admission-Controllers.md:113]:
```suggestion
  任何有权修改 webhook 对象的配置或准入控制器使用的工作负载的用户都可以破坏其运行。
```

File: content/zh/blog/_posts/2022-01-19-Securing-Admission-Controllers.md [hunk 12]
```diff
@@ -0,0 +1,156 @@
...
+  因此，确保只有集群管理员拥有这些权限非常重要。
+<!--
+* **Prevent privileged workloads**. One of the realities of container systems is that if a workload is given certain privileges, it will be possible to break out to the underlying cluster node and impact other containers on that node. Where admission controller services run in the cluster they’re protecting, it’s important to ensure that any requirement for privileged workloads is carefully reviewed and restricted as much as possible.
+-->  
+* **防止特权工作负载**。
+  容器系统的一个现实是，如果工作负载被赋予某些特权，
+  则有可能突破到底层集群节点并影响该节点上的其他容器。
```
Comment (tengqm) [inline content/zh/blog/_posts/2022-01-19-Securing-Admission-Controllers.md:120]:
```suggestion
  则有可能逃逸到下层的集群节点并影响该节点上的其他容器。
```

File: content/zh/blog/_posts/2022-01-19-Securing-Admission-Controllers.md [hunk 13]
```diff
@@ -0,0 +1,156 @@
...
+  如果准入控制器服务在它们所保护的集群上运行，
+  一定要确保对特权工作负载的任何要求都要经过仔细审查并尽可能地加以限制。
```
Comment (tengqm) [inline content/zh/blog/_posts/2022-01-19-Securing-Admission-Controllers.md:122]:
```suggestion
  一定要确保对特权工作负载的所有请求都要经过仔细审查并尽可能地加以限制。
```

File: content/zh/blog/_posts/2022-01-19-Securing-Admission-Controllers.md [hunk 14]
```diff
@@ -0,0 +1,156 @@
...
+<!--
+* **Strictly control external system access**. As a security service in a cluster admission controller systems will have access to sensitive information like credentials. To reduce the risk of this information being sent outside the cluster, [network policies](/docs/concepts/services-networking/network-policies/) should be used to restrict the admission controller services access to external networks.
+-->  
+* **严格控制外部系统访问**。
+  作为集群准入控制器系统中的安全服务，将有权访问敏感信息，如凭证。
```
Comment (tengqm) [inline content/zh/blog/_posts/2022-01-19-Securing-Admission-Controllers.md:127]:
```suggestion
  作为集群中的安全服务，准入控制器系统将有权访问敏感信息，如凭证。
```

File: content/zh/blog/_posts/2022-01-19-Securing-Admission-Controllers.md [hunk 15]
```diff
@@ -0,0 +1,156 @@
...
+  为了降低此信息被发送到集群外的风险，
+  应使用[网络策略](/zh/docs/concepts/services-networking/network-policies/) 
+  来限制准入控制器服务对外部网络的访问。
+<!--
+* **Each cluster has a dedicated webhook**. Whilst it may be possible to have admission controller webhooks that serve multiple clusters, there is a risk when using that model that an attack on the webhook service would have a larger impact where it’s shared. Also where multiple clusters use an admission controller there will be increased complexity and access requirements, making it harder to secure.
+-->  
+* **每个集群都有一个专用的 webhook**。
+  虽然可能有服务于多个集群的准入控制器 webhook，
```
Comment (tengqm) [inline content/zh/blog/_posts/2022-01-19-Securing-Admission-Controllers.md:135]:
```suggestion
  虽然可能让准入控制器 webhook 服务于多个集群，
```

File: content/zh/blog/_posts/2022-01-19-Securing-Admission-Controllers.md [hunk 16]
```diff
@@ -0,0 +1,156 @@
...
+
+<!--
+### Admission controller rules
+-->
+### 准入控制器规则
+
+<!--
+A key element of any admission controller used for Kubernetes security is the rulebase it uses. The rules need to be able to accurately meet their goals avoiding false positive and false negative results. 
+-->
+任何用于 Kubernetes 安全的准入控制器的一个关键元素是它使用的规则库。
```
Comment (tengqm) [inline content/zh/blog/_posts/2022-01-19-Securing-Admission-Controllers.md:147]:
```suggestion
对于用于 Kubernetes 安全的所有准入控制器而言，一个关键元素是它使用的规则库。
```

File: content/zh/blog/_posts/2022-01-19-Securing-Admission-Controllers.md [hunk 17]
```diff
@@ -0,0 +1,156 @@
...
+规则需要能够准确地满足其目标，避免误报和误报结果。
```
Comment (tengqm) [inline content/zh/blog/_posts/2022-01-19-Securing-Admission-Controllers.md:148]:
```suggestion
规则需要能够准确地满足其目标，避免假阳性和假阴性结果。
```

**Comment by twilight0620** [comment]
I have modified these comments, please review again, thanks.

**Comment by tengqm** [comment]
Thanks.
/lgtm
/approve

**Pull request merged**